### DISCUSSION TITLE
Error when running any command

### DISCUSSION THREAD

**Discussion by QuantaNeon** [FlawCra/ReactSelfbot#19] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/84242952/151208159-3f70250a-8c92-4893-bdfc-0aedaad13d95.png)
Right now i am running it on oracle linux i have also tired this with ubuntu and same error occured...pls tell me how to fix it

**Comment by cranberry3148** [comment]
1. ignore that. this only happens because you have not set up webhooks
2. try switchmode

**Comment by QuantaNeon** [comment]
> 1. ignore that. this only happens because you have not set up webhooks
> 2. try switchmode

1. there is also this error now that ive updated all webhook urls, also help command and stuff arent working
![image](https://user-images.githubusercontent.com/84242952/151290619-c9094475-df6b-4e57-a38a-f25e65732f38.png)

2. what is switchmode?

**Comment by MichaelDavidGK** [comment, marked as answer]
Embeds don't work with selfbots, since discord removed embeds (for non bots)

if you type your prefix and then switchmode for example >switchmode
then it will switch to codeblocks and will show messages in codeblocks
